release-notes-1.1.0.md

```diff
@@ -4,5 +4,7 @@
 ${version-number}
 
 #### New Features
+- **SCMOD-8480** - Updated to python [version 3.8.1](https://www.python.org/downloads/release/python-381/)
```

Comment:
Just say latest development version of Python in the release notes because this might change to 3.8.2 etc. without us making a change.